### ISSUE TITLE
compatibility with mobile user agents.

### ISSUE THREAD

**Issue by BuddhaDiedLaughing** [josStorer/chatGPTBox#305]
When using the plug-in on Firefox for Android I have to switch to desktop user agent for CGPT to appear which makes the rest of the page awful to view.

Please return functionality to mobile Firefox.  I believe it used to work.

Thank you for all you do.

**Comment by josStorer** [comment]
On the Kiwi browser, the automatic zoom function works fine. I will check the issue on Firefox Android.

**Comment by BuddhaDiedLaughing** [comment]
Thank you very much.  I'm so sorry in not in a position to donate right 
now.  I'll make a point of it something gives.  No BS.  I really appreciate 
what you're doing I've got this extension on every browser on every 
platform.  It's excellent

**Comment by josStorer** [comment]
Github's email notification doesn't seem to display the edited content correctly.

I'm not quite sure what you mean by CGPT that you mentioned. I would need further explanation about that.

**Comment by josStorer** [comment]
[v2.3.1](https://github.com/josStorer/chatGPTBox/releases/tag/v2.3.1) have improved mobile view

**Comment by BuddhaDiedLaughing** [comment]
Thank you.  Waiting for Firefox mobile to update to verify.  Incidentally 
if you know of a way to push it to update 'collections' I'm all ears.

**Comment by josStorer** [comment]
The addon updates for Firefox Mobile are synchronized with the desktop version. You just need to add it to your custom collections, and it will update automatically.
In addition, I cannot push it to the built-in addon list in Firefox Mobile. This list only allows official recommended addons.

**Comment by BuddhaDiedLaughing** [comment]
It's still not working for me with default user agent on Google.  Thanks 
for your efforts, though.

**Comment by BuddhaDiedLaughing** [comment]
When selecting to open as a floating box on the search results page in 
Firefox Android it opens fine but the icons at the top of the floating 
window are unresponsive.  Including the little X to close the box.

**Comment by PeterDaveHello** [comment]
CGPT = ChatGPT?

**Comment by BuddhaDiedLaughing** [comment]
Yes.  While I have you is it possible to incorporate GPT4FREE into the 
plug-in?  Especially on Android would be incredible.

Also, I generated an API # but can't figure out where it goes in settings.

Thanks!

**Comment by josStorer** [comment]
[v2.3.7](https://github.com/josStorer/chatGPTBox/releases/tag/v2.3.7)
